desktop/libs/notebook/src/notebook/connectors/hiveserver2.py

```diff
@@ -434,6 +434,109 @@ def get_jobs(self, notebook, snippet, logs):
 
     return jobs
 
+  def _get_fetch_result_spk(self, _statement, snippet):
```

Comment:
Could we move those below utilities functions (below line 730+ at least, where non API functions with _ starts)?

Reply:
Actually, could we move this to the beeswax.api modules?
(we hide HS2 implementation details like _autocomplete there)

Reply:
Hello ~ romainr ~

Reply:
Glad to receive your message, I'm tried make a encapsulation to move this api to "beewax.api".